tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -209,9 +344,18 @@
 			updateLink();
 		}
 
+		/**
+		 * Update the link.
+		 */
 		function updateLink() {
-			const link = getLink( currentBranch );
-			$( '#jetpack-beta-branch-link' ).attr( 'href', link ).text( link );
+			const $link = $( '#jetpack-beta-branch-link' );
+			const url = getLink();
+
+			if ( url.match( /[?&]branch(es\[[^&=]*\])?=/ ) ) {
+				$link.attr( 'href', url ).text( url );
+			} else {
+				$link.attr( 'href', null ).text( 'Select at least one plugin to test' );
```

Comment:
I see this message for this particular PR. 

It does make sense since no plugins are affected by this change, but maybe we can update the messaging to make it clearer?

Reply:
I'd be happy to take suggestions.